src/web/session-summary.jsp

```diff
@@ -95,99 +95,72 @@
     sessionResultFilter.setSortOrder(order);
     // Filter out the dodgy looking sessions
     // Note; dodgy looking cast as we're stuck using Java 7 to compile JSPs - see OF-1528
```

Comment:
This comment is no longer valid now, is it?

Reply:
You're right, yes.